app/controllers/permissions_controller.rb

```diff
@@ -1,19 +1,28 @@
 class PermissionsController < ApplicationController
   include FiltersHelper
   include TaxonomyHelper
-  respond_to :js
 
-  def index
-    type = params[:resource_type].presence
-    @permissions = Permission.where(:resource_type => type)
-    @search_path = search_path(type)
-    @granular = granular?(type)
-
-    if @granular
-      resource_class = Filter.get_resource_class(type)
-      @show_organizations = show_organization_tab? && resource_class.allows_organization_filtering?
-      @show_locations = show_location_tab? && resource_class.allows_location_filtering?
-    end
+  def show_resource_types_with_translations
```

Comment:
I think you will need to define a `action_permission` method which will map `params[:action]` from `show_resource_types_with_translations` to `view` to fix the tests.

Reply:
Thanks! I added it and the tests are still showing a permission error, any suggestions?

Reply:
Maybe changing routes.rb L281 from `only: [:index]` to `only: []`?

Reply:
It fixed the failing `permissions` test, thanks. Not sure new test failures are related